requirements.txt

```diff
@@ -1,11 +1,13 @@
+appdirs==1.4.4
+black==21.5b2
 bracex==2.1.1
 certifi==2020.12.5
 chardet==4.0.0
 click==7.1.2
 future==0.18.2
 graphkb==1.5.4
 idna==2.10
-importlib-metadata==4.0.1
+importlib-metadata==1.7.0
```

Comment:
this is weird, is it intentional?

Reply:
I saw that too, but It is a dependency of a dependency so I suspect that one of the other packages just required a lower version and the others previously hadn't cared